docs/downloads/automation-library/standard/pr-description-checklist/pr_description_checklist.cm

```diff
@@ -0,0 +1,21 @@
+# -*- mode: yaml -*-
+
+manifest:
+  version: 1.0
+
+automations:
+  pr_description_checklist:
+    if:
+      - {{ pr.description | includes(regex=r/\-\[\]/) }}
+    run:
+      - action: add-label@v1
+        args:
+          label: "Missing Checklist Items"
+          color: {{ colors.yellow }}
+      - action: request-changes@v1
+        args:
+          comment: |
+            Please complete the PR checklist before requesting a code review.
```

Comment:
How about doing 2 checks, one for optional checklist items and one for mandatory. We cna suggest that a check list item that marked as mandatory blocks PR with request change and that optionl one result in a `missing` label.

- [ ] optional
- [ ] mandatory *

for the baove mandatory regex is `r/\- \[ \].*\*$/`